src/libraries/System.Private.CoreLib/src/System/Reflection/Metadata/MetadataUpdateHandlerAttribute.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Reflection.Metadata
+{
+    /// <summary>Specifies a type that should receive notifications of metadata updates.</summary>
```

Comment:
It would help to document the expected shape for the handler type here (if this is what ultimately ends up in the API docs).

Reply:
> It would help to document the expected shape for the handler type here 

It would.  We don't know it yet :)